build/core/config.mk

```diff
@@ -14,6 +14,11 @@
 # limitations under the License.
 
 CM_SRC_API_DIR := $(TOPDIR)prebuilts/cmsdk/api
+INTERNAL_CM_PLATFORM_API_FILE := $(TARGET_OUT_COMMON_INTERMEDIATES)/PACKAGING/cm_public_api.txt
```

Comment:
This and reintroduction of build/tasks/apicheck.mk essentially undoes 07ddbbcb847f0deab0c0a6d7bbfc245995e72ef1. Halium does not utilize framework APIs in any way as it does not run Java runtime, but having those used to lead to build problems before.